meta-security-smack/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -15,6 +15,21 @@ _ System -wx---
 ^ System::Run rwxat-
 ^ System rwxa--
 EOF
+
+    mkdir -p ${D}/${libdir}/tmpfiles.d
+    cat >${D}/${libdir}/tmpfiles.d/packet-forwarding.conf <<EOF
+t /proc/sys/net/ipv4/conf/all/forwarding - - - - security.SMACK64=*
+t /proc/sys/net/ipv6/conf/all/forwarding - - - - security.SMACK64=*
+t /proc/sys/net/ipv4/conf/default/forwarding - - - - security.SMACK64=*
+t /proc/sys/net/ipv6/conf/default/forwarding - - - - security.SMACK64=*
+EOF
+
```

Comment:
Alrighty... But again, for consistency, the same should be done for ${D}/${sysconfdir}/smack/accesses.d.